src/script/InitDelegateeDeposits.s.sol

```diff
@@ -0,0 +1,152 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+import {Script} from "forge-std/Script.sol";
+import {console2} from "forge-std/console2.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {Multicall} from "@openzeppelin/contracts/utils/Multicall.sol";
+import {IGovLst, Staker} from "../interfaces/IGovLst.sol";
+
+/// @title InitDelegateeDeposits
+/// @author [ScopeLift](https://scopelift.co)
+/// @notice A script to initialize deposits for delegatee addresses in the GovLst contract.
+/// @dev This script reads delegatee addresses from a JSON file, checks if they already have deposits,
+/// and initializes deposits for those that don't have one yet using multicall batching for efficiency.
+abstract contract InitDelegateeDeposits is Script {
+  /// @notice Reference to the GovLst contract.
+  IGovLst govLst = IGovLst(getGovLstAddress());
```

Comment:
Why not have the virtual `getGovLstAddress` return the correct type, rather than casting here?